pkg/apis/workflow/v1alpha1/cron_workflow_types.go

```diff
@@ -42,6 +42,8 @@ type CronWorkflowSpec struct {
 	FailedJobsHistoryLimit *int32 `json:"failedJobsHistoryLimit,omitempty" protobuf:"varint,7,opt,name=failedJobsHistoryLimit"`
 	// Timezone is the timezone against which the cron schedule will be calculated, e.g. "Asia/Tokyo". Default is machine's local time.
 	Timezone string `json:"timezone,omitempty" protobuf:"bytes,8,opt,name=timezone"`
+	// WorkflowMetadata contains some metadata of the workflow to be run
+	WorkflowMetadata *metav1.ObjectMeta `json:"workflowMetadata,omitempty" protobuf:"bytes,9,opt,name=workflowMeta"`
```

Comment:
rename `metadata`?

Reply:
I would like `workflowMetadata` to be analogous to `wokflowSpec` so that it's clear that both fields' data is meat for the child `Workflow`

Reply:
so this is different to the example for deployment

Reply:
Yes, it currently works different to Deployment. I mention that in my previous comment:

> `CronWorkflow` works like this latter example (`workflowSpec`), so to be able to specify labels and annotations a new field is needed (`workflowMetadata`). Another option is to rework how the templates are defined to be more analogous to `Deployment -> Pod` or `CronJob -> Job`; i.e. put the whole template under template: such a change would be backwards compatible.